### PR TITLE
Temp

### PR DESCRIPTION
**Pull request by sharmoo**
Removed sensitive files

### CODE REVIEW

File: backendAS/tables.sql [hunk 1]
```diff
@@ -3,9 +3,8 @@ create database player;
 
 \connect player
 
-
 create table song(
-  id int serial primary key,
+  id serial primary key,
```
Comment (pafrias) [inline backendAS/tables.sql:7]: Could you tell me what serial does over int serial?

File: server/db.js [hunk 2]
```diff
@@ -1,197 +1,165 @@
 const { Pool } = require('pg');
-const { user, password } = require('./config');
+const { user, password } = require('./config.js');
 
 const pool = new Pool({
-
   user: user,
-  host: '',
+  host: process.env.DB_HOST || '',
```
Comment (pafrias) [inline server/db.js:6]: Nice use of environment variables

File: server/db.js [hunk 3]
```diff
@@ -1,197 +1,165 @@
...
+          client.release();
+
+          return resp.rows;
+        })
+        .catch(err => {
+          client.release()
+          return err.stack;
+        })
+    })
 }
```
Comment (pafrias) [inline server/db.js:161]: Very nice use of promises

**Review by pafrias — APPROVED** [review]
Nice work, well formatted promises, proper number of files for a PR.

File: backendAS/tables.sql (hunk 1 above)
Comment (sharmoo) [reply backendAS/tables.sql:7]: word on the street is that serial results in the automatic creation of an int or bigint column.....
so no need to specify.

File: server/db.js (hunk 3 above)
Comment (sharmoo) [reply server/db.js:161]: <3

Comment (sharmoo) [reply server/db.js:161]: Wonder where those came from.....

**Pull request merged**